src/components/Home/BookForm.css

```diff
@@ -18,16 +18,20 @@ form > .form-container {
 
 form > h3 {
   margin-bottom: 10px;
+  color: #888;
 }
 
 .input {
   padding: 0.813rem 2.688rem;
   margin-right: 2rem;
+  border: 1px solid #e8e8e8;
 }
 
 .category {
+  border: 1px solid #e8e8e8;
   padding: 0.375rem 9.938rem 0.563rem 1rem;
   margin-right: 2rem;
+  color: grey;
 }
```

Comment:
- [ ] I would recommend using the DRY principle here. 

```css

.input, .category {
  border: ....;
  margin-right: 2rem;
}

.input {
  ...
  ...
}

.category {
  ...
  ...
}
```

Reply:
Thanks for pointing that out :+1: